### ISSUE TITLE
Add default values for accessibility if HF is non-functioning

### ISSUE THREAD

**Issue by luna404** [HeRAMS-WHO/herams-backend#434]
Requested by Yemen, if a facility is not functioning, accessibility should always default to not-functional. 

To implement this in LS the following steps will be required

- [ ] add a new answer option "Not functioning" to HFACC 
- [ ] use relevance to display answer option in HFACC.  "Not Functioning" should only be visible if HFFUNCT is set to "not functioning". All other answer options for HFACC should have the inverse logic, meaning they only show if the HF is functioning.
- [ ] set the default value for HFACC to "not functioning". (I'm not sure this will actually work, but I'm hoping this will only add a default value if "not functioning" is relevant. If it doesn't work, we can try and use the same logic used to add default values for array questions).
- [ ] condition HFACCx to not display if HFACC is set to "not functioning".

**Comment by JeffDotingco** [comment]
Hi @luna404, I have already created a survey to check the possibility of a non-functioning response in HFACC. It seems to work if HFFUNCT=="Non-functioning". However the default values of HFFUNCT as "non-functioning" if CONDB=="fully damaged" from an equation is not working. I think Yemen IM also raised this issue.

Kindly check if you are free. :)

**Comment by luna404** [comment]
took me a little while to understand what you were doing... I guess my idea of using relevance is not working for single choice question so you are trying to use a hidden array filter. Is this correct?

A few points I noticed / not fully understand

1. Why do you have equation to set the default values for HFFUNCT and HFACC? Can we not use the standard way of defining default values?
2. HFFUNCT alawys default to `not functioning`. Is this an error or are you doing this on prupose? I though it should only default on `not functioning if the HF is destroyed. 
3. If I set HFFUNCT to `not functioning` I don't get any answer options. Is this a mistake in the hidden array question?

More generally, I think we need to re-discuss our options here. @JeffDotingco your solution (if we get it to work correctly) would definitely be the nicest one from a data entry perspective. However, it will create a lot of unused fields in the data export and I'm worried that this will confuse users even more. If we are starting to make surveys more complex (and I'm definitely also guilty of doing this), could we discuss always hiding certain fields from the front-end export?
Instead of using the array filter, we could also simply the entire process and always display all answer options. Users could be forced to enter the correct value using validation formulas. It's not as nice from a data entry perspective but it would avoid creating the hidden questions. 

On a side note, @JeffDotingco I can't go back to a previous page in the survey. Do you have any idea why and could this be fixed. It would make testing a lot easier :)

**Comment by JeffDotingco** [comment]
Yes, agree @luna404 . I sill couldn't get HFFUNCT to be "Non-functioning" when it is destroyed. For item no. 3, the responses for HFACC (accessible, partially accessible, not accessible) are hidden if HFFUNCT==Not functioning, but the corresponding answer for HFACC is "not functioning". That is still the issue I am trying to solve. :)

**Comment by luna404** [comment]
@JeffDotingco I had another look at the test survey you created. I fixed the issue with "Not functioning" not appearing for "HFACC". I also changed the behaviour for CONDB/HFFUNCT slightly. I though it might be better to use the same logic there, meaning if a HF is destroyed HFFUNCT will default to "Not functioning". One of the question that we would still have to address is how to deal with the default value for HFFUNCT (I don't like using `not functioning` as a default) and HFFUNCTx.


@SamuelPetragallo it seems LS is a little more fuzzy than we anticipated. @JeffDotingco found a solution to have a behavior similar to what we discussed  [link](https://ls.herams.org/552633?newtest=Y) , but in my opinion there are a few drawbacks to this. First, it requires hidden multiple choice question, which will appear in the data export. Second, we display questions to user that they cannot alter (LS does not allow to hide them), which is weird from a user perspective. 
For me the issue is more a data analysis issue and it would be better if the issue were addressed there. I think we have discussed this a couple of months ago but I don't remember the conclusion. Could we use an equation question at the the end of the survey that defines whether `Unkown` is truly `Unkown` or another value. For example, if a HF is `not functioning` the equation result would be `not functioning`. In the dashboard, instead of having the HF displayed as `unkonwn` it would now show `not functioning`. 

If addressing addressing the issue at the data analysis side is not an option, I think we could implement @JeffDotingco solution **if** we have an option to always exclude certain fields from the front-end data export.
